design-library/source/_patterns/00-global/01-base-colors.twig

```diff
@@ -0,0 +1,35 @@
+<ul class="sg-colors">
```

Comment:
should there be a "01-base-colors" directory for this pattern? Should this and the other "01-base-colors" files be in the proposed directory?